### PR TITLE
Einav add build

### PR DESCRIPTION
**Pull request by nushicoco**

### CODE REVIEW

**Comment by nushicoco** [comment]
just adding build + postgres updates

File: db/reset_db.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
 const { User, FrequentProblem } = require('../models')
 
 User.sync({force: true})
-FrequentProblem.sync({force: true})
+FrequentProblem.sync({force: true}).then(function () {
+
+    FrequentProblem.destroy({truncate: true})
+        .then( () => frequentProblemValues.forEach( ([env, subEnv, subject, solution, solutionURL]) => {
+            FrequentProblem.create({env, subEnv, subject, solutionURL }).then(function(user) {
+                // you can now access the newly created task via the variable task
```
Comment (dancar) [inline db/reset_db.js]: ?

File: db/reset_db.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+                console.log('success');
+            }).catch(function(err) {
+                // print the error details
+                console.log(err, env, subEnv, subject, solutionURL );
```
Comment (dancar) [inline db/reset_db.js]: Aren't these variables out of the catch scope?

File: models/index.js [hunk 3]
```diff
@@ -1,6 +1,9 @@
 const Sequelize = require('sequelize')
 const dbPath = process.env.DATABASE_PATH || 'db/winder.db'
-const sequelize = new Sequelize(`sqlite://${dbPath}`, {
+const dbURL = process.env.DATABASE_URL || `sqlite://${dbPath}`;
```
Comment (dancar) [inline models/index.js]: Maybe we can simply get rid of `dbPath`altogether and use the full descriptor (`sqllite://db/winder.db`) as a default value for `dbURL` ?

File: models/index.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
...
+// const dbURL = "postgres://pjugqmxrwtegqj:fd4f702fa8bf07cc24342d65ed06be2213cb11a40294af117b218c4725c79633@ec2-107-20-186-238.compute-1.amazonaws.com:5432/d3fh9c39bc2rna"
+// const dbURL = "postgres://nushi:Gf3O0ncr8Suy@localhost:5432/einavcarmon"
```
Comment (dancar) [inline models/index.js]: 😶

File: db/reset_db.js [hunk 5]
```diff
@@ -1,7 +1,19 @@
 const { User, FrequentProblem } = require('../models')
 
 User.sync({force: true})
-FrequentProblem.sync({force: true})
+FrequentProblem.sync({force: true}).then(function () {
+
+    FrequentProblem.destroy({truncate: true})
```
Comment (nushicoco) [inline db/reset_db.js]: yeah force is drop, I will remove the destroy

File: db/reset_db.js (hunk 2 above)
Comment (nushicoco) [inline db/reset_db.js]: I copied from a stackoverflow solution so we can see where the error occurred, so i'm not sure. . I can remove it for now.

File: models/index.js (hunk 3 above)
Comment (nushicoco) [inline models/index.js]: done.

File: db/reset_db.js (hunk 5 above)
Comment (nushicoco) [inline db/reset_db.js]: update - something is weird for the reset_db with postgres.. logged it on trello

**Pull request merged**